src/Sylius/Behat/Context/Ui/Admin/BrowsingProductVariantsContext.php

```diff
@@ -74,6 +74,8 @@ public function theProductVariantShouldNotAppearInTheShop($productVariantCode, P
 
     /**
      * @Then the :product product should have no variants
+     * @Given (this product) should have no variants
```

Comment:
`@Given` -> `@Then`

Reply:
`(this product) should have no variants` -> `/^(this product) should have no variants$/`